Example/Example/ControlsDemoView.swift

```diff
@@ -0,0 +1,61 @@
+// Created by Cal Stephens on 8/11/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import Lottie
+import SwiftUI
+
+// MARK: - ControlsDemoView
+
+struct ControlsDemoView: View {
+
+  var body: some View {
+    List {
+      LottieSwitchRow(
+        animationName: "Samples/Switch",
+        title: "Switch",
+        onTimeRange: 0.5...1.0,
+        offTimeRange: 0.0...0.5)
+
+      LottieSwitchRow(
+        animationName: "Samples/Switch",
+        title: "Switch (Custom Colors)",
+        onTimeRange: 0.5...1.0,
+        offTimeRange: 0.0...0.5,
+        colorValueProviders: [
+          "Checkmark Outlines.Group 1.Stroke 1.Color": [Keyframe(.black)],
+          "Checkmark Outlines 2.Group 1.Stroke 1.Color": [Keyframe(.black)],
+          "X Outlines.Group 1.Stroke 1.Color": [Keyframe(.black)],
+          "Switch Outline Outlines.Fill 1.Color": [
+            Keyframe(value: LottieColor.black, time: 0),
+            Keyframe(value: LottieColor(r: 0.76, g: 0.76, b: 0.76, a: 1), time: 75),
+            Keyframe(value: LottieColor.black, time: 150),
+          ],
+        ])
+
+      // TODO: Add SwiftUI support for AnimatedButton
+      // AnimatedButtonRow(
+      //   animationName: "Samples/TwitterHeartButton",
+      //   title: "Twitter Heart Button",
+      //   playRanges: [
+      //     .init(fromMarker: "touchDownStart", toMarker: "touchDownEnd", event: .touchDown),
+      //     .init(fromMarker: "touchDownEnd", toMarker: "touchUpCancel", event: .touchUpOutside),
+      //     .init(fromMarker: "touchDownEnd", toMarker: "touchUpEnd", event: .touchUpInside),
+      //   ]))
```

Comment:
Should we remove the comments until we have the demo working?

Reply:
Nah I'll uncomment this out in the next PR